### PR TITLE
feat(banned): describe additional ban capabilities

### PR DESCRIPTION
**Pull request by savonarola**
Docs for [EMQX-11769](https://emqx.atlassian.net/browse/EMQX-11769), [EMQX-11786](https://emqx.atlassian.net/browse/EMQX-11786), https://github.com/emqx/emqx/pull/12499

This should be merged after the dashboard is also updated (https://emqx.atlassian.net/browse/ED-1238).

### CODE REVIEW

File: en_US/access-control/blacklist.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 EMQX provides users with a banning functionality to deny the access from a specific client. You can add a client to the banned list by the client ID, user name or source IP address.
 
+Additionally, clients may be banned by rules based on:
+* matching client identifiers and usernames against a regular pattern;
+* matching source IP addresses against a CIDR range.
+
+::: warning
```
Comment (zmstone) [inline en_US/access-control/blacklist.md]:
```suggestion
::: tip
```

File: en_US/access-control/blacklist.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
 
 EMQX provides users with a banning functionality to deny the access from a specific client. You can add a client to the banned list by the client ID, user name or source IP address.
 
+Additionally, clients may be banned by rules based on:
```
Comment (zmstone) [inline en_US/access-control/blacklist.md]:
```suggestion
Additionally, clients may be banned by matching rules based on:
```

**Comment by zmstone** [comment]
LGTM @Meggielqk  please help to translate.

**Comment by Meggielqk** [comment]
@ysfscream, please help to check if the new "Banned Object" terms match those on the Dashboard UI.

**Comment by ysfscream** [comment]
@Meggielqk Sure, I will update this today.

**Comment by ysfscream** [comment]
The dashboard updated on: https://github.com/emqx/emqx-dashboard5/pull/2421

**Review by zmstone — APPROVED** [review]

**Comment by zmstone** [comment]
sorry, did not notice the merge-base was wrong. 
will revert this and @savonarola  please send a new one to `release-5.6`

**Pull request merged**